.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Python tests
+
+on:
+  push:
+    branches: 'master'
+  pull_request:
+    branches: 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
This is normally not necessary, unless you're facing specific issues. Also, try not to dump all the commands in a single step because it's harder to process the test job run report visually then.

Reply:
ok